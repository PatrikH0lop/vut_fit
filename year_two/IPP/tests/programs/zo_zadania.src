.IPPcode18
DEFVAR GF@counter
MOVE GF@counter string@a
LABEL while
JUMPIFEQ end GF@counter string@aaa
WRITE string@counter_obsahuje
WRITE GF@counter
CONCAT GF@counter GF@counter string@a
JUMP while
LABEL end
WRITE string@counter_obsahuje
WRITE GF@counter
