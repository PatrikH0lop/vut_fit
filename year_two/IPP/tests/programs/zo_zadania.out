counter_obsahuje
a
counter_obsahuje
aa
counter_obsahuje
aaa
